app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+namespace Suyabay\Http\Controllers;
+
+use Suyabay\User;
+use Suyabay\Role;
+use Suyabay\Invite;
+use Suyabay\Http\Requests;
+use Illuminate\Http\Request;
+use Illuminate\Mail\Mailer as Mail;
+use Suyabay\Http\Controllers\Controller;
+
+class UserController extends Controller
+{
+    protected $mail;
+    protected $response;
+
+    public function __construct(Mail $mail)
+    {
+        $this->mail = $mail;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $users = User::orderBy('id', 'asc')->paginate(10);
+
+        return view('dashboard.pages.user', compact('users'));
+    }
+
+    /**
+     * Insert invitation details to invites tablee
+     *
+     * @param  Request $request
+     */
+    public function createInvite(Request $request)
+    {
+        $createInvite = Invite::firstOrCreate([
+            'username'  => $request->username,
+            'role_id'  => $request->user_role,
+            'token'   => $request->_token
+        ]);
+        if ($createInvite)
```

Comment:
There should be newline space between line 46 and 47 and also space before your final `return`